pkg/tests/end_to_end_tests/ingest_trace_test.go

```diff
@@ -207,6 +207,35 @@ func findTraceTest(t testing.TB, q *query.Query) {
 	require.NoError(t, err)
 	require.Equal(t, 1, len(traces))
 
+	// Verify that we find traces by status code based on translated tag.
+	request = &spanstore.TraceQueryParameters{
+		ServiceName:   "service-name-0",
+		OperationName: "operationA",
+		Tags: map[string]string{
+			"error": "true",
```

Comment:
Consider testing other tags like `kind` and `space`. Also worth testing `error = false`